python/lsst/ts/dsm/utils.py

```diff
@@ -0,0 +1,75 @@
+import csv
+import os
+import yaml
+
+from astropy.time import Time, TimeDelta
+import numpy as np
+import pytz
+
+__all__ = ['convert_time', 'create_telemetry_config', 'create_telemetry_data']
+
+
+def convert_time(in_time):
+    """Convert an ISO UTC timestring to TAI timestamp.
```

Comment:
FYI: I expanded salobj tai_from_utc to accept UTC in any format accepted by AstroPy. But it won't be available until release of salobj v5 (with SAL v4). You could copy the code for now if you want to avoid the leap second issue mentioned below. Or switch before the next leap second.